make/mt-config/mt-tls-wolfssl.mk

```diff
@@ -28,6 +28,7 @@ IOTC_CONFIG_FLAGS += -DHAVE_ECC
 IOTC_CONFIG_FLAGS += -DTFM_TIMING_RESISTANT -DECC_TIMING_RESISTANT -DWC_RSA_BLINDING
 IOTC_CONFIG_FLAGS += -DUSE_FAST_MATH
 IOTC_CONFIG_FLAGS += -DWOLFSSL_X86_64_BUILD
+#TODO(frec): define the SHA512 and ed5139 stuff here ?
```

Comment:
These preprocessor defines define how the Device SDK sees the wolfssl API, i.e. they affect only the wolfssl API, not the library functionality itself. Wolfssl lib content is defined by `res/tls/wolfssl.conf`. 
In theory this list should match with `AM_CFLAGS` in file `third_party/tls/wolfssl/config.log`, which is generated during wolfssl build. In practice only API affecting flags have to be defined.